### ISSUE TITLE
App crashes when answering a call from app killed state when using out-of-band Firebase Messaging service on Android

### ISSUE THREAD

**Issue by aaronleo28** [twilio/twilio-voice-react-native#445]
# Issue

## Pre-submission Checklist
- [x] I have verified that the issue occurs with the latest release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-voice-react-native/blob/main/CHANGELOG.md).
- [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice-react-native/blob/main/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
- [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

## Description 

I'm building an Expo react native app that uses this SDK alongside our own `@react-native-firebase/messaging` service that I use to receive push notifications sent from our backend.

I was happy to see the 1.2.1 release go out which unblocked us from using firebase messaging in our app for Android with the introduction of support for an out-of-band Firebase service.

I set this up following [this guide](https://github.com/twilio/twilio-voice-react-native/blob/main/docs/out-of-band-firebase-messaging-service.md) and everything works well except responding to calls made while the app is killed and the screen is off.

Calls made while the app is foregrounded or backgrounded work as expected, the issue only occurs when trying to answer a call via the notification presented when the app is killed and the screen is off.

## Reproduction Steps

1. Add an out of band Firebase service as per the docs
2. fully close the app and turn off the screen
3. make a call to the device
4. try answer the call by pressing the notification
5. see the app crash

## Expected Behavior
I would expect the same behaviour that is observed when not using an out of band Firebase service, ie that a full-screen accept/reject call view is presented, as well as the incoming call notification, which can be used to answer the call successfully

## Actual Behavior

The device will call, and a notification can be seen, but pressing the notification tries to open the app which will then crash. Also, the full screen accept/reject call view isn't visible

## Reproduction Frequency

100%

## Software and Device Information
**Please complete the following information.**
 - Device: Samsung Galaxy S10e
 - OS: Android 12
 - React version: 18.2.0
 - React Native version: 0.74.5
 - Node version: 20.11.0
 - npm or yarn version: 1.22.19
 - expo version: 51.0.34
 - react-native-firebase version: 20.4.0

**Comment by mhuynh5757** [comment]
Hi @aaronleo28 thanks for bringing this to our attention. I indeed tested the "app killed" use-case before release, so let's dig into this and see if we can resolve the issue. Could you get the logcats and post any errors you see here?

**Comment by aaronleo28** [comment]
@mhuynh5757 thanks for the speedy reply, I've included the logcats below, filtered for warnings and above

btw, the behaviour is as expected when the screen is on and the app is killed - the issue only arises when the app is killed and the screen is off

```
11-05 14:09:23.802 11865 11865 E USNET   : USNET: appName: com.example.exampleapp
11-05 14:09:23.936 11865 11865 W SoLoader: Recording new base apk path: /data/app/~~5WqSM2cqaz-kqTfs-AFxrQ==/com.example.exampleapp-jbIBdWzXWNHg1_BeluTNoQ==/base.apk
11-05 14:09:23.936 11865 11865 W SoLoader: Previously recorded 0 base apk paths.
11-05 14:09:25.348  1095  4742 W ActivityTaskManager: startActivity called from non-Activity context; forcing Intent.FLAG_ACTIVITY_NEW_TASK for: Intent { act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION cmp=com.example.exampleapp/.MainActivity (has extras) }
11-05 14:09:25.528 11865 11865 E AndroidRuntime: FATAL EXCEPTION: main
11-05 14:09:25.528 11865 11865 E AndroidRuntime: Process: com.example.exampleapp, PID: 11865
11-05 14:09:25.528 11865 11865 E AndroidRuntime: java.lang.RuntimeException: Unable to pause activity {com.example.exampleapp/com.example.exampleapp.MainActivity}: java.lang.reflect.InvocationTargetException
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:5557)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.ActivityThread.performPauseActivity(ActivityThread.java:5508)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:5460)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.servertransaction.PauseActivityItem.execute(PauseActivityItem.java:47)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: Caused by: java.lang.reflect.InvocationTargetException
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at expo.modules.ReactActivityDelegateWrapper.invokeDelegateMethod(ReactActivityDelegateWrapper.kt:328)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at expo.modules.ReactActivityDelegateWrapper.onPause(ReactActivityDelegateWrapper.kt:203)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at com.facebook.react.ReactActivity.onPause(ReactActivity.java:53)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.Activity.performPause(Activity.java:8485)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnPause(Instrumentation.java:1530)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:5547)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	... 14 more
11-05 14:09:25.528 11865 11865 E AndroidRuntime: Caused by: java.lang.AssertionError
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at com.facebook.infer.annotation.Assertions.assertCondition(Assertions.java:82)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at com.facebook.react.ReactInstanceManager.onHostPause(ReactInstanceManager.java:597)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at com.facebook.react.ReactDelegate.onHostPause(ReactDelegate.java:134)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	at com.facebook.react.ReactActivityDelegate.onPause(ReactActivityDelegate.java:134)
11-05 14:09:25.528 11865 11865 E AndroidRuntime: 	... 21 more
11-05 14:09:25.537  1095  4229 W ActivityManager: crash : com.example.exampleapp,10347
11-05 14:09:25.538  1095  4229 W ActivityTaskManager:   Force finishing activity com.example.exampleapp/.MainActivity
11-05 14:09:25.635  1095  4742 W InputManager-JNI: Input channel object 'b02ec4a com.example.exampleapp/com.example.exampleapp.MainActivity (client)' was disposed without first being removed with the input manager!
11-05 14:09:25.635  1095  2714 W ActivityManager: Scheduling restart of crashed service com.example.exampleapp/com.twiliovoicereactnative.VoiceService in 1000ms for start-requested
11-05 14:09:25.644  1095  2714 W InputManager-JNI: Input channel object '95c9daf Splash Screen com.example.exampleapp (client)' was disposed without first being removed with the input manager!
11-05 14:09:26.700 12007 12007 E USNET   : USNET: appName: com.example.exampleapp
11-05 14:09:26.957 12007 12007 W SoLoader: Recording new base apk path: /data/app/~~5WqSM2cqaz-kqTfs-AFxrQ==/com.example.exampleapp-jbIBdWzXWNHg1_BeluTNoQ==/base.apk
11-05 14:09:26.957 12007 12007 W SoLoader: Previously recorded 0 base apk paths.
11-05 14:09:26.994 12007 12007 E AndroidRuntime: FATAL EXCEPTION: main
11-05 14:09:26.994 12007 12007 E AndroidRuntime: Process: com.example.exampleapp, PID: 12007
11-05 14:09:26.994 12007 12007 E AndroidRuntime: java.lang.RuntimeException: Unable to start service com.twiliovoicereactnative.VoiceService@7a3ee44 with Intent { act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION cmp=com.example.exampleapp/com.twiliovoicereactnative.VoiceService (has extras) }: java.lang.NullPointerException
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5110)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at android.app.ActivityThread.access$2200(ActivityThread.java:310)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2319)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: Caused by: java.lang.NullPointerException
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at java.util.Objects.requireNonNull(Objects.java:220)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at com.twiliovoicereactnative.VoiceService.getCallRecord(VoiceService.java:390)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at com.twiliovoicereactnative.VoiceService.onStartCommand(VoiceService.java:135)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5092)
11-05 14:09:26.994 12007 12007 E AndroidRuntime: 	... 9 more
11-05 14:09:26.999  1095  2026 W ActivityManager: crash : com.example.exampleapp,10347
11-05 14:09:27.079  1095 15363 W ActivityManager: Scheduling restart of crashed service com.example.exampleapp/com.twiliovoicereactnative.VoiceService in 1800000ms for start-requested
11-05 14:09:31.322  1095  4742 W ActivityTaskManager: startActivity called from non-Activity context; forcing Intent.FLAG_ACTIVITY_NEW_TASK for: Intent { act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION cmp=com.example.exampleapp/.MainActivity (has extras) }
11-05 14:09:31.375 12141 12141 E USNET   : USNET: appName: com.example.exampleapp
11-05 14:09:31.451 12141 12141 W SoLoader: Recording new base apk path: /data/app/~~5WqSM2cqaz-kqTfs-AFxrQ==/com.example.exampleapp-jbIBdWzXWNHg1_BeluTNoQ==/base.apk
11-05 14:09:31.451 12141 12141 W SoLoader: Previously recorded 0 base apk paths.
11-05 14:09:31.506 12141 12141 E AndroidRuntime: FATAL EXCEPTION: main
11-05 14:09:31.506 12141 12141 E AndroidRuntime: Process: com.example.exampleapp, PID: 12141
11-05 14:09:31.506 12141 12141 E AndroidRuntime: java.lang.RuntimeException: Unable to start service com.twiliovoicereactnative.VoiceService@4912a0f with Intent { act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION cmp=com.example.exampleapp/com.twiliovoicereactnative.VoiceService (has extras) }: java.lang.NullPointerException
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5110)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at android.app.ActivityThread.access$2200(ActivityThread.java:310)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2319)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: Caused by: java.lang.NullPointerException
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at java.util.Objects.requireNonNull(Objects.java:220)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at com.twiliovoicereactnative.VoiceService.getCallRecord(VoiceService.java:390)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at com.twiliovoicereactnative.VoiceService.onStartCommand(VoiceService.java:135)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5092)
11-05 14:09:31.506 12141 12141 E AndroidRuntime: 	... 9 more
11-05 14:09:31.508  1095  2773 W ActivityManager: crash : com.example.exampleapp,10347
11-05 14:09:31.509  1095  2773 W ActivityTaskManager:   Force finishing activity com.example.exampleapp/.MainActivity
11-05 14:09:32.011  1095  1455 W ActivityTaskManager: Activity top resumed state loss timeout for ActivityRecord{9cd942f u0 com.example.exampleapp/.MainActivity t316 f}}
11-05 14:09:32.022  1095  1455 W ActivityTaskManager: Activity pause timeout for ActivityRecord{9cd942f u0 com.example.exampleapp/.MainActivity t316 f}}
11-05 14:09:32.036  1095  4742 W InputManager-JNI: Input channel object '9c8d60f Splash Screen com.example.exampleapp (client)' was disposed without first being removed with the input manager!
11-05 14:09:32.040  1095  1456 E WindowManager: win=Window{9c8d60f u0 Splash Screen com.example.exampleapp EXITING} destroySurfaces: appStopped=false win.mWindowRemovalAllowed=true win.mRemoveOnExit=true win.mViewVisibility=8 caller=com.android.server.wm.ActivityRecord.destroySurfaces:5848 com.android.server.wm.ActivityRecord.destroySurfaces:5829 com.android.server.wm.WindowState.onExitAnimationDone:5975 com.android.server.wm.ActivityRecord$$ExternalSyntheticLambda13.accept:2 java.util.ArrayList.forEach:1262 com.android.server.wm.ActivityRecord.onAnimationFinished:7897 com.android.server.wm.ActivityRecord.postApplyAnimation:5579
11-05 14:09:42.054  1095  1455 W ActivityTaskManager: Activity destroy timeout for ActivityRecord{9cd942f u0 com.example.exampleapp/.MainActivity t316 f}}
11-05 14:09:42.055  1095  1455 W InputManager-JNI: Input channel object '13f2656 com.example.exampleapp/com.example.exampleapp.MainActivity (client)' was disposed without first being removed with the input manager!
11-05 14:09:45.315  1095  1470 W BroadcastQueue: Receiver during timeout of BroadcastRecord{d894472 u0 com.google.android.c2dm.intent.RECEIVE} : ResolveInfo{7028d1b com.example.exampleapp/io.invertase.firebase.messaging.ReactNativeFirebaseMessagingReceiver m=0x108000}
11-05 14:09:55.319  1095  1470 W BroadcastQueue: Receiver during timeout of BroadcastRecord{d894472 u0 com.google.android.c2dm.intent.RECEIVE} : ResolveInfo{55f7156 com.example.exampleapp/com.google.firebase.iid.FirebaseInstanceIdReceiver m=0x108000}
11-05 14:09:55.325 24420 24420 W GCM     : broadcast intent callback: result=CANCELLED forIntent { act=com.google.android.c2dm.intent.RECEIVE flg=0x10000000 pkg=com.example.exampleapp (has extras) }
11-05 14:10:01.493  1095  1470 W ActivityManager: Timeout executing service: ServiceRecord{7f60232 u0 com.example.exampleapp/com.twiliovoicereactnative.VoiceService}
```

**Comment by mhuynh5757** [comment]
Thanks! We'll look into it further.

**Comment by aaronleo28** [comment]
happy friday @mhuynh5757 are there any updates on the above issue?

**Comment by greenraze13** [comment]
@aaronleo28 HI , Have you found any solution for this ?

**Comment by aaronleo28** [comment]
@greenraze13 Hey, unfortunately not :(

We're completely blocked on this, so we're using expo-notifications instead, but even with this there are issues receiving notifications on Android platform

Ideally we'd use firebase messaging as there are more features available, but this issue makes this unavailable for us

**Comment by dwbelliston** [comment]
Okay, also seeing this issue.

What is working:
- App is killed, but screen on
- Call comes in, click notification, all works

What is broken
- App is killed, lock the screen
- Call comes in, app crashes

Here are my logcat logs:


```
2025-02-07 10:23:50.863 27453-27595 ReactNativeJNI          com.currentclient.app                I  Memory warning (pressure level: TRIM_MEMORY_BACKGROUND) received by JS VM, running a GC
2025-02-07 10:23:55.329 27453-27595 ReactNativeJNI          com.currentclient.app                I  Memory warning (pressure level: TRIM_MEMORY_BACKGROUND) received by JS VM, running a GC
2025-02-07 10:23:55.341  1829-3856  PowerManagerService     system_server                        I  Allowing device wake-up without android.permission.TURN_SCREEN_ON for com.currentclient.app
2025-02-07 10:23:55.369 27453-27453 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioManagerExternal: Sample rate is set to 48000 Hz
2025-02-07 10:23:55.369 27453-27453 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioManagerExternal: Sample rate is set to 48000 Hz
2025-02-07 10:23:55.370 27453-27453 tvo.webrtc.Logging      com.currentclient.app                I  JavaAudioDeviceModule: createAudioDeviceModule
2025-02-07 10:23:55.370 27453-27453 tvo.webrtc.Logging      com.currentclient.app                I  JavaAudioDeviceModule: HW NS will be used.
2025-02-07 10:23:55.370 27453-27453 tvo.webrtc.Logging      com.currentclient.app                I  JavaAudioDeviceModule: HW AEC will be used.
2025-02-07 10:23:55.370 27453-27453 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioEffectsExternal: ctor@[name=main, id=2]
2025-02-07 10:23:55.370 27453-27453 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioRecordExternal: ctor@[name=main, id=2]
2025-02-07 10:23:55.370 27453-27453 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioTrackExternal: ctor@[name=main, id=2]
2025-02-07 10:23:55.399 27453-28290 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioRecordExternal: enableBuiltInAEC(true)
2025-02-07 10:23:55.399 27453-28290 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioEffectsExternal: setAEC(true)
2025-02-07 10:23:55.400 27453-28290 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioRecordExternal: enableBuiltInNS(true)
2025-02-07 10:23:55.400 27453-28290 tvo.webrtc.Logging      com.currentclient.app                I  WebRtcAudioEffectsExternal: setNS(true)
2025-02-07 10:23:55.423 27453-27453 Twilio:cla...lang.Class com.currentclient.app                I  [Platform] {"timestamp_ms":1738949035415,"client_name":"43714665_2f6e_44c5_bd43_b49038f78263","call_sid":"CA16ee2ca4ac14642e3b111e0d3d22f688","message_sid":"RUab638ad097954b6806711d6a007f74da","sdk_version":"1.3.0","platform":"android","direction":"INCOMING","selected_region":"gll","logger_type":"default"}
2025-02-07 10:23:55.432 27453-27453 VoiceFireb...ingService com.currentclient.app                I  onCallInvite CA16ee2ca4ac14642e3b111e0d3d22f688
2025-02-07 10:23:55.452  1829-4773  MediaFocusControl       system_server                        I  requestAudioFocus() from uid/pid 10339/27453 AA=USAGE_VOICE_COMMUNICATION/CONTENT_TYPE_SPEECH clientId=android.media.AudioManager@a6ff862com.twilio.audioswitch.AudioSwitch$1@a6ef8ca callingPack=com.currentclient.app req=2 flags=0x1 sdk=34
2025-02-07 10:23:55.457 27453-27453 Twilio:cla...lang.Class com.currentclient.app                I  [Platform] {"timestamp_ms":1738949035456,"client_name":"43714665_2f6e_44c5_bd43_b49038f78263","call_sid":"CA16ee2ca4ac14642e3b111e0d3d22f688","message_sid":"RUab638ad097954b6806711d6a007f74da","sdk_version":"1.3.0","platform":"android","direction":"INCOMING","selected_region":"gll","logger_type":"default"}
2025-02-07 10:23:55.501 27453-27595 ReactNativeJNI          com.currentclient.app                I  Memory warning (pressure level: TRIM_MEMORY_BACKGROUND) received by JS VM, running a GC
2025-02-07 10:23:55.665  1829-4067  ActivityTaskManager     system_server                        W  startActivity called from non-Activity context; forcing Intent.FLAG_ACTIVITY_NEW_TASK for: Intent { act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION cmp=com.currentclient.app/.MainActivity (has extras) }
2025-02-07 10:23:55.670  2595-2631  WindowManagerShell      com.android.systemui                 V  Transition requested (#886): android.os.BinderProxy@913dc04 TransitionRequestInfo { type = OPEN, triggerTask = TaskInfo{userId=0 taskId=938 effectiveUid=10339 displayId=0 isRunning=true baseIntent=Intent { act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION flg=0x10000000 cmp=com.currentclient.app/.MainActivity } baseActivity=ComponentInfo{com.currentclient.app/com.currentclient.app.MainActivity} topActivity=ComponentInfo{com.currentclient.app/com.currentclient.app.MainActivity} origActivity=null realActivity=ComponentInfo{com.currentclient.app/com.currentclient.app.MainActivity} numActivities=1 lastActiveTime=146013417 supportsMultiWindow=true resizeMode=1 isResizeable=true minWidth=-1 minHeight=-1 defaultMinSize=220 token=WCT{android.window.IWindowContainerToken$Stub$Proxy@99615ed} topActivityType=1 pictureInPictureParams=null shouldDockBigOverlays=false launchIntoPipHostTaskId=-1 lastParentTaskIdBeforePip=-1 displayCutoutSafeInsets=Rect(0, 132 - 0, 0) topActivityInfo=ActivityInfo{1e3bc22 com.currentclient.app.MainActivity} launchCookies=[] positionInParent=Point(0, 0) parentTaskId=-1 isFocused=true isVisible=false isVisibleRequested=false isSleeping=true locusId=null displayAreaFeatureId=1 isTopActivityTransparent=false isTopActivityStyleFloating=false lastNonFullscreenBounds=Rect(276, 694 - 804, 1774) capturedLink=null capturedLinkTimestamp=0 appCompatTaskInfo=AppCompatTaskInfo { topActivityInSizeCompat=false eligibleForLetterboxEducation= false isLetterboxEducationEnabled= false isLetterboxDoubleTapEnabled= false eligibleForUserAspectRatioButton= false topActivityBoundsLetterboxed= false isFromLetterboxDoubleTap= false topActivityLetterboxVerticalPosition= -1 topActivityLetterboxHorizontalPosition= -1 topActivityLetterboxWidth=1080 topActivityLetterboxHeight=2400 topActivityLetterboxAppWidth=1080 topActivityLetterboxAppHeight=2205 isUserFullscreenOverrideEnabled=false isSystemFullscreenOverrideEnabled=false hasMinAspectRatioOverride=false cameraCompatTaskInfo=CameraCompatTaskInfo { freeformCameraCompatMode=inactive}}}, pipTask = null, remoteTransition = null, displayChange = null, flags = 0, debugId = 886 }
2025-02-07 10:23:55.671  1829-4067  ActivityTaskManager     system_server                        I  START u0 {act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION cmp=com.currentclient.app/.MainActivity (has extras)} with LAUNCH_SINGLE_TASK from uid 10339 (realCallingUid=10229) (BAL_ALLOW_VISIBLE_WINDOW) result code=0
2025-02-07 10:23:55.703 27453-27453 ContentCaptureHelper    com.currentclient.app                I  Setting logging level to OFF
2025-02-07 10:23:55.714  1829-3856  CoreBackPreview         system_server                        D  Window{391b9e3 u0 Splash Screen com.currentclient.app}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@10ba409, mPriority=0, mIsAnimationCallback=false}
2025-02-07 10:23:55.717 27453-27453 TransactionExecutor     com.currentclient.app                E  Failed to execute the transaction: tId:-202321297 ClientTransaction{
                                                                                                    tId:-202321297   transactionItems=[
                                                                                                    tId:-202321297     TopResumedActivityChangeItem{mActivityToken=android.os.BinderProxy@74a13c0,onTop=true}
                                                                                                    tId:-202321297     Target activity: com.currentclient.app.MainActivity
                                                                                                    tId:-202321297     TopResumedActivityChangeItem{mActivityToken=android.os.BinderProxy@74a13c0,onTop=false}
                                                                                                    tId:-202321297     Target activity: com.currentclient.app.MainActivity
                                                                                                    tId:-202321297     PauseActivityItem{mActivityToken=android.os.BinderProxy@74a13c0,finished=false,userLeaving=true,dontReport=false,autoEnteringPip=false}
                                                                                                    tId:-202321297     Target activity: com.currentclient.app.MainActivity
                                                                                                    tId:-202321297   ]
                                                                                                    tId:-202321297 }
2025-02-07 10:23:55.717 27453-27453 AndroidRuntime          com.currentclient.app                D  Shutting down VM
2025-02-07 10:23:55.718 27453-27453 AndroidRuntime          com.currentclient.app                E  FATAL EXCEPTION: main
                                                                                                    Process: com.currentclient.app, PID: 27453
                                                                                                    java.lang.RuntimeException: Unable to pause activity {com.currentclient.app/com.currentclient.app.MainActivity}: java.lang.reflect.InvocationTargetException
                                                                                                    	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:5689)
                                                                                                    	at android.app.ActivityThread.performPauseActivity(ActivityThread.java:5640)
                                                                                                    	at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:5592)
                                                                                                    	at android.app.servertransaction.PauseActivityItem.execute(PauseActivityItem.java:60)
                                                                                                    	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:63)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.executeLifecycleItem(TransactionExecutor.java:169)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:101)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:80)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2719)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:109)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:232)
                                                                                                    	at android.os.Looper.loop(Looper.java:317)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8787)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:591)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:871)
                                                                                                    Caused by: java.lang.reflect.InvocationTargetException
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at expo.modules.ReactActivityDelegateWrapper.invokeDelegateMethod(ReactActivityDelegateWrapper.kt:328)
                                                                                                    	at expo.modules.ReactActivityDelegateWrapper.onPause(ReactActivityDelegateWrapper.kt:203)
                                                                                                    	at com.facebook.react.ReactActivity.onPause(ReactActivity.java:53)
                                                                                                    	at android.app.Activity.performPause(Activity.java:9224)
                                                                                                    	at android.app.Instrumentation.callActivityOnPause(Instrumentation.java:1778)
                                                                                                    	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:5679)
                                                                                                    	... 15 more
                                                                                                    Caused by: java.lang.AssertionError
                                                                                                    	at com.facebook.infer.annotation.Assertions.assertCondition(Assertions.java:82)
                                                                                                    	at com.facebook.react.ReactInstanceManager.onHostPause(ReactInstanceManager.java:597)
                                                                                                    	at com.facebook.react.ReactDelegate.onHostPause(ReactDelegate.java:134)
                                                                                                    	at com.facebook.react.ReactActivityDelegate.onPause(ReactActivityDelegate.java:134)
                                                                                                    	... 22 more
2025-02-07 10:23:55.720  1829-2210  ActivityTaskManager     system_server                        W    Force finishing activity com.currentclient.app/.MainActivity
2025-02-07 10:23:55.722 27453-27453 Process                 com.currentclient.app                I  Sending signal. PID: 27453 SIG: 9
2025-02-07 10:23:55.761  1829-2209  WindowManager           system_server                        I  WIN DEATH: Window{9ca825b u0 com.currentclient.app/com.currentclient.app.MainActivity}
2025-02-07 10:23:55.761  1829-2522  ConnectivityService     system_server                        D  releasing NetworkRequest [ REQUEST id=1407, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED&NOT_VCN_MANAGED&NOT_BANDWIDTH_CONSTRAINED Uid: 10339 RequestorUid: 10339 RequestorPkg: com.currentclient.app UnderlyingNetworks: Null] ] (release request)
2025-02-07 10:23:55.761  1829-2522  ConnectivityService     system_server                        D  releasing NetworkRequest [ REQUEST id=1409, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED&NOT_VCN_MANAGED&NOT_BANDWIDTH_CONSTRAINED Uid: 10339 RequestorUid: 10339 RequestorPkg: com.currentclient.app UnderlyingNetworks: Null] ] (release request)
2025-02-07 10:23:55.763  1829-1949  WindowManager           system_server                        V  Sent Transition (#886) createdAt=02-07 10:23:55.665 via request=TransitionRequestInfo { type = OPEN, triggerTask = TaskInfo{userId=0 taskId=938 effectiveUid=10339 displayId=0 isRunning=true baseIntent=Intent { act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION flg=0x10000000 cmp=com.currentclient.app/.MainActivity } baseActivity=ComponentInfo{com.currentclient.app/com.currentclient.app.MainActivity} topActivity=ComponentInfo{com.currentclient.app/com.currentclient.app.MainActivity} origActivity=null realActivity=ComponentInfo{com.currentclient.app/com.currentclient.app.MainActivity} numActivities=1 lastActiveTime=146013417 supportsMultiWindow=true resizeMode=1 isResizeable=true minWidth=-1 minHeight=-1 defaultMinSize=220 token=WCT{RemoteToken{e4c8e27 Task{d7acf86 #938 type=standard A=10339:com.currentclient.app isResizeable=true minWidth=-1 minHeight=-1}}} topActivityType=1 pictureInPictureParams=null shouldDockBigOverlays=false launchIntoPipHostTaskId=-1 lastParentTaskIdBeforePip=-1 displayCutoutSafeInsets=Rect(0, 132 - 0, 0) topActivityInfo=ActivityInfo{31298d4 com.currentclient.app.MainActivity} launchCookies=[] positionInParent=Point(0, 0) parentTaskId=-1 isFocused=true isVisible=false isVisibleRequested=false isSleeping=true locusId=null displayAreaFeatureId=1 isTopActivityTransparent=false isTopActivityStyleFloating=false lastNonFullscreenBounds=Rect(276, 694 - 804, 1774) capturedLink=null capturedLinkTimestamp=0 appCompatTaskInfo=AppCompatTaskInfo { topActivityInSizeCompat=false eligibleForLetterboxEducation= false isLetterboxEducationEnabled= false isLetterboxDoubleTapEnabled= false eligibleForUserAspectRatioButton= false topActivityBoundsLetterboxed= false isFromLetterboxDoubleTap= false topActivityLetterboxVerticalPosition= -1 topActivityLetterboxHorizontalPosition= -1 topActivityLetterboxWidth=1080 topActivityLetterboxHeight=2400 topActivityLetterboxAppWidth=1080 topActivityLetterboxAppHeight=2205 isUserFullscreenOverrideEnabled=false isSystemFullscreenOverrideEnabled=false hasMinAspectRatioOverride=false cameraCompatTaskInfo=CameraCompatTaskInfo { freeformCameraCompatMode=inactive}}}, pipTask = null, remoteTransition = null, displayChange = null, flags = 0, debugId = 886 }
2025-02-07 10:23:55.763  1829-2210  ActivityManager         system_server                        I  Process com.currentclient.app (pid 27453) has died: fg  SVC 
2025-02-07 10:23:55.764  1829-2210  ActivityManager         system_server                        W  Scheduling restart of crashed service com.currentclient.app/com.twiliovoicereactnative.VoiceService in 1000ms for start-requested
2025-02-07 10:23:56.220  1829-1952  ActivityTaskManager     system_server                        W  Activity top resumed state loss timeout for ActivityRecord{6418861 u0 com.currentclient.app/.MainActivity t-1 f}}
2025-02-07 10:23:56.793  1829-1964  ActivityManager         system_server                        I  Start proc 28357:com.currentclient.app/u0a339 for service {com.currentclient.app/com.twiliovoicereactnative.VoiceService}
2025-02-07 10:23:56.825 28357-28357 nativeloader            com.currentclient.app                D  Load libframework-connectivity-tiramisu-jni.so using APEX ns com_android_tethering for caller /apex/com.android.tethering/javalib/framework-connectivity-t.jar: ok
2025-02-07 10:23:56.831  1829-2913  WindowManager           system_server                        V  Binding proc com.currentclient.app with config {1.0 ?mcc0mnc [en_US] ldltr sw411dp w411dp h840dp 420dpi nrml long hdr widecg port night finger -keyb/v/h -nav/h winConfig={ mBounds=Rect(0, 0 - 1080, 2400) mAppBounds=Rect(0, 132 - 1080, 2337) mMaxBounds=Rect(0, 0 - 1080, 2400) mDisplayRotation=ROTATION_0 mWindowingMode=fullscreen mActivityType=undefined mAlwaysOnTop=undefined mRotation=ROTATION_0} as.2 s.10 fontWeightAdjustment=0}
2025-02-07 10:23:56.834 28357-28357 CompatChangeReporter    com.currentclient.app                D  Compat change id reported: 242716250; UID 10339; state: ENABLED
2025-02-07 10:23:56.841 28357-28357 ziparchive              com.currentclient.app                W  Unable to open '/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.arm64_v8a.dm': No such file or directory
2025-02-07 10:23:56.841 28357-28357 ziparchive              com.currentclient.app                W  Unable to open '/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.arm64_v8a.dm': No such file or directory
2025-02-07 10:23:56.841 28357-28357 rrentclient.app         com.currentclient.app                W  Entry not found
2025-02-07 10:23:56.842 28357-28357 ziparchive              com.currentclient.app                W  Unable to open '/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.en.dm': No such file or directory
2025-02-07 10:23:56.842 28357-28357 ziparchive              com.currentclient.app                W  Unable to open '/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.en.dm': No such file or directory
2025-02-07 10:23:56.842 28357-28357 rrentclient.app         com.currentclient.app                W  Entry not found
2025-02-07 10:23:56.842 28357-28357 ziparchive              com.currentclient.app                W  Unable to open '/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.xxhdpi.dm': No such file or directory
2025-02-07 10:23:56.842 28357-28357 ziparchive              com.currentclient.app                W  Unable to open '/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.xxhdpi.dm': No such file or directory
2025-02-07 10:23:56.842 28357-28357 rrentclient.app         com.currentclient.app                W  Entry not found
2025-02-07 10:23:56.843 28357-28357 nativeloader            com.currentclient.app                D  Configuring clns-7 for other apk /data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/base.apk:/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.arm64_v8a.apk:/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.en.apk:/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.xxhdpi.apk. target_sdk_version=34, uses_libraries=, library_path=/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/lib/arm64:/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/base.apk!/lib/arm64-v8a:/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.arm64_v8a.apk!/lib/arm64-v8a:/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/split_config.en.apk!/lib/arm64-v8a:/data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4
2025-02-07 10:23:56.848 28357-28357 GraphicsEnvironment     com.currentclient.app                V  Currently set values for:
2025-02-07 10:23:56.848 28357-28357 GraphicsEnvironment     com.currentclient.app                V    angle_gl_driver_selection_pkgs=[com.android.angle, com.linecorp.b612.android, com.campmobile.snow, com.google.android.apps.tachyon]
2025-02-07 10:23:56.848 28357-28357 GraphicsEnvironment     com.currentclient.app                V    angle_gl_driver_selection_values=[angle, native, native, native]
2025-02-07 10:23:56.848 28357-28357 GraphicsEnvironment     com.currentclient.app                V  com.currentclient.app is not listed in per-application setting
2025-02-07 10:23:56.848 28357-28357 GraphicsEnvironment     com.currentclient.app                V  Neither updatable production driver nor prerelease driver is supported.
2025-02-07 10:23:56.869 28357-28357 FirebaseApp             com.currentclient.app                I  Device unlocked: initializing all Firebase APIs for app [DEFAULT]
2025-02-07 10:23:56.875 28357-28357 FirebaseInitProvider    com.currentclient.app                I  FirebaseApp initialization successful
2025-02-07 10:23:56.896 28357-28357 SoLoader                com.currentclient.app                W  Initializing SoLoader: 0
2025-02-07 10:23:56.896 28357-28357 SoLoader                com.currentclient.app                W  Recording new base apk path: /data/app/~~qShUrPxUCnS7C7dlqnsjIg==/com.currentclient.app-iPADAQxVQlmoc4zJUPmQ4A==/base.apk
                                                                                                    Previously recorded 0 base apk paths.
2025-02-07 10:23:56.900 28357-28357 SoLoader                com.currentclient.app                W  SoLoader initialized: 8
2025-02-07 10:23:56.903 28357-28376 MediaCodecList          com.currentclient.app                D  codecHandlesFormat: no format, so no extra checks
2025-02-07 10:23:56.903 28357-28375 MediaCodecList          com.currentclient.app                D  codecHandlesFormat: no format, so no extra checks
2025-02-07 10:23:56.905 28357-28357 BluetoothAdapter        com.currentclient.app                D  HEADSET connected
2025-02-07 10:23:56.909 28357-28378 CCodec                  com.currentclient.app                D  allocate(c2.android.raw.decoder)
2025-02-07 10:23:56.910 28357-28380 CCodec                  com.currentclient.app                D  allocate(c2.android.raw.decoder)
2025-02-07 10:23:56.911 28357-28378 Codec2Client            com.currentclient.app                I  Available Codec2 services: "default" "default1" "software"
2025-02-07 10:23:56.913 28357-28357 AndroidRuntime          com.currentclient.app                D  Shutting down VM
2025-02-07 10:23:56.913 28357-28378 CCodec                  com.currentclient.app                I  setting up 'default' as default (vendor) store
2025-02-07 10:23:56.913 28357-28380 CCodec                  com.currentclient.app                I  setting up 'default' as default (vendor) store
2025-02-07 10:23:56.914 28357-28357 AndroidRuntime          com.currentclient.app                E  FATAL EXCEPTION: main
                                                                                                    Process: com.currentclient.app, PID: 28357
                                                                                                    java.lang.RuntimeException: Unable to start service com.twiliovoicereactnative.VoiceService@2f4b783 with Intent { act=ACTION_FOREGROUND_AND_DEPRIORITIZE_INCOMING_CALL_NOTIFICATION cmp=com.currentclient.app/com.twiliovoicereactnative.VoiceService (has extras) }: java.lang.NullPointerException
                                                                                                    	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5246)
                                                                                                    	at android.app.ActivityThread.-$$Nest$mhandleServiceArgs(Unknown Source:0)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2546)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:109)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:232)
                                                                                                    	at android.os.Looper.loop(Looper.java:317)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8787)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:591)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:871)
                                                                                                    Caused by: java.lang.NullPointerException
                                                                                                    	at java.util.Objects.requireNonNull(Objects.java:207)
                                                                                                    	at com.twiliovoicereactnative.VoiceService.getCallRecord(VoiceService.java:390)
                                                                                                    	at com.twiliovoicereactnative.VoiceService.onStartCommand(VoiceService.java:135)
                                                                                                    	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:5228)
                                                                                                    	... 9 more
2025-02-07 10:23:56.918 28357-28357 Process                 com.currentclient.app                I  Sending signal. PID: 28357 SIG: 9
2025-02-07 10:23:56.957  1829-2913  ActivityManager         system_server                        I  Process com.currentclient.app (pid 28357) has died: fg  SVC 
```

**Comment by mhuynh5757** [comment]
Hey all, thanks for the additional crash log. I'm not sure if the original issue is related to the new crash, but let me dig into this considering the new info you've provided.

**Comment by mhuynh5757** [comment]
Internal ticket: https://twilio-engineering.atlassian.net/browse/VBLOCKS-4178

**Comment by tirth0** [comment]
Are these internal tickets public for us to track?

**Comment by mhuynh5757** [comment]
@tirth0 they are not, just for the Twilio SDK Engineering team to help keep track of which JIRA tickets map to which Github issues.

While I'm posting actually, we just released version `1.5.0` of the RN SDK which addresses some race conditions. Whilst I cannot confirm that it is the root cause of this Github issue, it may help solve some crashing problems. Please pull the latest version at your earliest convenience.

**Comment by mhuynh5757** [comment]
Hey all, going to close this one as stale for now. Please update to the latest SDK at your earliest convenience. If you are still encountering this issue after updating, please post a new Github Issue and we will address it. Thanks!
